api/libproject/devstart.rb

```diff
@@ -10,21 +11,95 @@ def dev_up()
   common.run_inline_swallowing_interrupt %W{docker-compose logs -f api}
 end
 
-def rebuild_image()
+def rebuild_image(args)
   common = Common.new
   common.docker.requires_docker
 
   common.run_inline %W{docker-compose build}
 end
 
+def auth_login(account)
+  common = Common.new
+  common.run_inline %W{gcloud auth login #{account}}
+end
+
+def set_project(project)
+  common = Common.new
+  common.run_inline %W{gcloud config set project #{project}}
+end
+
+def copy_file_to_gcs(source_path, bucket, filename)
+  common = Common.new
+  common.run_inline %W{gsutil cp #{source_path} gs://#{bucket}/#{filename}}
+end
+
+def create_db_creds(args)
+  usage = "Usage: --project <project> --account <account>"
+  common = Common.new()
+  if args.has_key?("project") && args.has_key?("account")
+    project = args["project"]
+    account = args["account"]
+    if project == nil || account == nil
+      raise(usage)
+    end
+    puts "Enter the root DB user password:"
```

Comment:
We could generate passwords here, but it may be useful to have the passwords stored in a vault externally (we're doing this in Valentine today.) That way if GCS is down or something we can still get at the database.